package.json

```diff
@@ -20,6 +20,7 @@
 		"lint": "eslint --ext .js,.vue src",
 		"lint:fix": "eslint --ext .js,.vue src --fix",
 		"test": "jest --verbose",
+		"test:coverage": "jest --verbose --coverage --no-cache",
```

Comment:
Is this necessary?

Reply:
I used `make test-coverage` and noticed this command doesn't work because the script `test:coverage` wasn't present so i fixed this. It's not necessary but i think it's nice if the Makefile stages work :-)